docs/design/ovs-pipeline.md

```diff
@@ -1,1197 +1,1938 @@
-additional dedicated tables to support the [Antrea-native policies](../antrea-network-policy.md)
-([AntreaPolicyEgressRuleTable] and [AntreaPolicyIngressRuleTable]).
+### LoadBalancer
 
-Consider the following Antrea ClusterNetworkPolicy (ACNP) in the Application tier as an
-example for the remainder of this document.
+A sample LoadBalancer Service with the ingress IP `192.168.77.150` assigned by an ingress controller.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: sample-loadbalancer
+spec:
+  selector:
+    app: web
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+  type: LoadBalancer
+status:
+  loadBalancer:
+    ingress:
+      - ip: 192.168.77.150
+```
+
+### LoadBalancer with Session Affinity
+
+A sample LoadBalancer Service with the ingress IP `192.168.77.151` assigned by an ingress controller,
+configured with session affinity.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: sample-loadbalancer-session-affinity
+spec:
+  selector:
+    app: web
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+  type: LoadBalancer
+  sessionAffinity: ClientIP
+  sessionAffinityConfig:
+    clientIP:
+      timeoutSeconds: 300
+status:
+  loadBalancer:
+      ingress:
+        - ip: 192.168.77.151
+```
+
+### Service with ExternalIP
+
+A sample Service with the external IP `192.168.77.200`.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: sample-service-externalip
+spec:
+  selector:
+    app: web
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+  externalIPs: 
+    - 192.168.77.200
+```
+
+### Service with ExternalIP and Session Affinity
+
+A sample Service with the external IP `192.168.77.201` configured with session affinity.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: sample-service-externalip-session-affinity
+spec:
+  selector:
+    app: web
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+  externalIPs:
+    - 192.168.77.201
+  sessionAffinity: ClientIP
+  sessionAffinityConfig:
+    clientIP:
+      timeoutSeconds: 300
+```
+
+## Antrea-native NetworkPolicy Implementation
+
+In addition to the tables created for Kubernetes NetworkPolicy, Antrea creates additional dedicated tables to support
+the [Antrea-native NetworkPolicy](../antrea-network-policy.md) (table [AntreaPolicyEgressRule] and [AntreaPolicyIngressRule]).
+
+Consider the following Antrea ClusterNetworkPolicy (ACNP) in the Application tier as an example for the remainder of
+this document.
+
+This ACNP is applied to all Pods with the label `app: web` in all Namespaces. For these Pods, only TCP traffic on port
+80 from the Pods with the label `app: client` and to the Pods with the label `app: db` is allowed. Similar to Kubernetes
+NetworkPolicy, Antrea will only install OVS flows for this policy on Nodes that have Pods selected by the policy. We
+still use the Pods with the label `app: web` mentioned above as the target of this policy.
```

Comment:
I would add a quick note here to avoid creating confusion for the reader:

> This policy has very similar rules as the K8s NetworkPolicy example shown previously. This is intentional to simplify this document and to allow easier comparison between the flows generated for both types of policies. Additionally, we should emphasize that this policy applies to Pods across all Namespaces, while a K8s NetworkPolicy is always scoped to a specific Namespace (in the case of our example, the `default` Namespace).

Reply:
Thanks for the note. I have added it.